charts/bindplane/templates/transform-agent.yaml

```
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: {{ include "bindplane.deployment_type" . }}
```

```suggestion
kind: deployment
```

BindPlane supports deployment and statefulset, and is a statefulset by default. The transform agent should probably run as a deployment.